src/Migration/Migration1603384890AdyenPaymentResponse.php

```diff
@@ -0,0 +1,58 @@
+<?php declare(strict_types=1);
+
+namespace Adyen\Shopware\Migration;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Migration\MigrationStep;
+
+class Migration1603384890AdyenPaymentResponse extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1603384890;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $connection->executeUpdate("
+            alter table `adyen_payment_response`
+            add column order_transaction_id binary(16) after sales_channel_api_context_token;
+        ");
+        $connection->executeUpdate("
+            update `adyen_payment_response` apr
+                inner join `order` o on apr.order_number = o.order_number
+                inner join `order_transaction` ot on o.id = ot.order_id and o.version_id = ot.order_version_id
+            set apr.order_transaction_id = ot.id
+            where 1=1;
+        ");
+        $connection->executeUpdate("
+            alter table `adyen_payment_response`
+                drop column order_number,
+                drop column sales_channel_api_context_token;
+        ");
```

Comment:
Should we move this update to updateDestructive since it is dropping columns?

Reply:
If I understood correctly, `update` is called when upgrading the plugin and `upgradeDestructive` is called when downgrading it.